Dockerfile

```diff
@@ -0,0 +1,43 @@
+# Installation stage
+# Offical Node.js Alpine base image
+FROM node:lts-alpine AS base
+
+# Set working directory
+WORKDIR /base
+
+# Copy package.json and package-lock.json for utilising Docker cache 
+# to save re-installing dependencies if unchanged
+COPY package*.json yarn.lock ./ 
+
+# Install dependencies only needed
+RUN yarn install --frozen-lockfile
+
+# Copy installed dependencies to current working directory
+COPY . .
+
+# Build stage
+FROM base AS builder
+
+# TO DO in further development
+# ENV NODE_ENV=production
+
+WORKDIR /build
+COPY --from=base /base .
```

Comment:
Since you add a comment about this on `base` and `production` stage. We should add a comment for this too.

Reply:
If you mean the comment about `ENV NODE_ENV`, in this file I add in `build` and `production` stage.

Reply:
Nope, I mean to add this `# Copy files from base stage to builder stage`

Reply:
Okay, I'll add for more clarification.